src/components/AvailableRockCard/index.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import * as gql from "../../queries";
+import { fetchData } from "../../utils";
+import { Redirect } from 'react-router-dom'
+
+export class AvailableRockCard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      createdRelationship: false
+    };
+  }
+
+  createList = (skills) => {
+    return skills.map((skill, i) => {
+      return (
+        <li key={i} className="AvailableRockCard-li">
+          {skill}
+        </li>
+      );
+    });
+  };
+
+  handleClick = async () => {
+    const { rock, user } = this.props;
+    try {
+      const body = gql.createRockPebbleRelationship(rock, user);
+      const response = await fetchData(body);
+      console.log(response)
+      if (response.createRockPebbleRelationship) {
+        this.setState({ createdRelationship: true });
+      } else {
+        //     setAvailableRocks([]);
+        //     this.setState({
```

Comment:
Did you mean to have an error message here?

Reply:
whoops! Nope. :D